content/en/docs/reference/using-api/server-side-apply.md

```diff
@@ -459,7 +459,7 @@ non-default field manager, as seen in the following example. The default field
 manager for kubectl server-side apply is `kubectl`.
 
 ```shell
-kubectl apply --server-side --field-manager=my-manager [--dry-run=server]
+kubectl apply --field-manager=my-manager [--dry-run=server]
```

Comment:
@apelisse , Do you have time to review this change? Thank you.

Reply:
Yeah, I still don't know if I'm convinced at least that this command is better.

The text above reads:
> This behavior applies to server-side apply with the `kubectl` field manager.
As an exception, you can opt-out of this behavior by specifying a different,
non-default field manager, as seen in the following example. The default field
manager for kubectl server-side apply is `kubectl`.

"if you want to server-side apply the non-default manager, you can do ..."
and then the command is right. The whole thing might be confusing or not helpful, 
but AFAIU the command is exactly what the text says.